docs/DEVELOPMENT.md

```diff
@@ -52,10 +55,50 @@ After making your changes go in the deltachat/electron Dev-console and press `F5
 
 > **Note:** this only applies to the frontend code in src/renderer. To build the main process you still need to use `npm run build` and then restart the deltachat-desktop process. (`npm run start`)
 
+### Code Style <a id="code-style"></a>
+
+#### Linting <a id="linting"></a>
+
+We use [eslint](https://eslint.org) with typescript rules to catch common bad-practices.
+
+Commands:
+
+```sh
+# Run to test
+npm run lint
+# fix autofixable problems and fix formatting
+npm run lint:fix-format
```

Comment:
how about `npm run lint:fix` ? I think keeping those things short is good.

Reply:
also formats the document... I kinda wanted to include what it does